packages/store/test/StoreCore.t.sol

```diff
@@ -503,6 +513,111 @@ contract StoreCoreTest is Test, StoreView {
     assertEq(keccak256(loadedData), keccak256(new bytes(schema.staticDataLength())));
   }
 
+  function testPushToField() public {
+    uint256 table = uint256(keccak256("some.table"));
+
+    {
+      // Register table's schema
+      Schema schema = SchemaLib.encode(SchemaType.UINT256, SchemaType.UINT32_ARRAY, SchemaType.UINT32_ARRAY);
+      StoreCore.registerSchema(table, schema);
+    }
```

Comment:
I haven't seen this pattern before. Do the curly braces/block create a new scope? So `schema` is only available inside the scope?

Reply:
> Do the curly braces/block create a new scope?

yes, afaik it can also be useful outside of tests to avoid stack limits with many vars